template/module/static/src/js/module_name.tour.js

```diff
@@ -0,0 +1,54 @@
+/* © <YEAR(S)> <AUTHOR(S)>
```

Comment:
This is great, what about an inheritance template? https://github.com/odoo/odoo/blob/9.0/addons/website_sale_options/static/src/js/website_sale.test.js#L9